packages/components/src/button/src/Button.css

```diff
@@ -78,159 +78,213 @@ a.o-ui-button {
     min-height: var(--o-ui-button-height-sm);
 }
 
+.o-ui-button-sm .o-ui-button-text {
+    font-size: var(--hop-body-sm-medium-font-size);
+}
+
 .o-ui-button-md {
     min-height: var(--o-ui-button-height-md);
 }
 
+.o-ui-button-md .o-ui-button-text {
+    font-size: var(--hop-body-md-medium-font-size);
+}
+
 /* PRIMARY */
 .o-ui-button-primary {
-    color: var(--o-ui-text-alias-static-white);
-    background-color: var(--o-ui-bg-alias-accent);
+    background-color: var(--hop-primary-surface-strong);
+    color: var(--hop-primary-text-strong);
 }
 
 /* PRIMARY | HOVER */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):hover,
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading).o-ui-button-hover {
-    background-color: var(--o-ui-bg-alias-accent-hover);
+    background-color: var(--hop-primary-surface-strong-hover);
 }
 
 /* PRIMARY | HOVER | ACTIVE */
 .o-ui-button-primary:not([disabled]):not(.o-ui-button-loading):hover:active,
 .o-ui-button-primary:not([disabled]).o-ui-button-hover.o-ui-button-active {
-    background-color: var(--o-ui-bg-alias-accent-active);
+    background-color: var(--hop-primary-surface-strong-active);
+    outline: 1px solid var(--hop-primary-border-active);
+    color: var(--hop-primary-text-active);
 }
 
 /* PRIMARY | FOCUS */
 .o-ui-button-primary:not([disabled]):focus-visible,
 .o-ui-button-primary:not([disabled]).o-ui-button-focus {
-    background-color: var(--o-ui-bg-alias-accent);
+    background-color: var(--hop-primary-surface-hover);
 }
 
 /* PRIMARY | HOVER | FOCUS */
 .o-ui-button-primary:hover:focus-visible,
 .o-ui-button-primary.o-ui-button-hover.o-ui-button-focus {
-    background-color: var(--o-ui-bg-alias-accent-hover);
+    background-color: var(--hop-primary-surface-strong-hover);
```

Comment:
After clicking the secondary and tertiary buttons, The focus background color remains:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/60e3428d-4a19-4454-a636-42a03741e4a8)
This should have the same behavior as the blue outline and only show when using the keyboard.